client/keys/add.go

```diff
@@ -136,7 +141,6 @@ func RunAddCmd(ctx client.Context, cmd *cobra.Command, args []string, inBuf *buf
 			}
 		}
 
-		multisigKeys, _ := cmd.Flags().GetStringSlice(flagMultisig)
 		if len(multisigKeys) != 0 {
 			var pks []cryptotypes.PubKey
```

Comment:
Instead of `pks = append(pks, k.GetPubKey())` you can paste `pks[i] = k.GetPubKey()` . That will resolve the panic at line 162.